### ISSUE TITLE
Transparency shows overlap

### ISSUE THREAD

**Issue by italodeandra** [feathericons/feather#1001]
## Prerequisites

* Version: 4.28.0
* Running from: Plain SVG files
* Operating system: Windows 10
* Bits: 64

## Step to reproduce

- Add a transparent color to the icon
- Notice the overlapping

### Actual behavior

- Add a transparent color to the icon
- It should not have overlapping

## Any message or error

No console output

## Resources

![image](https://user-images.githubusercontent.com/19225266/98681690-503f6b00-2363-11eb-93bf-b6d54fe4e3e4.png)

**Comment by yukosgiti** [comment]
Have you tried to change the `opacity` of the icon instead of the `transperancy`?

**Comment by italodeandra** [comment]
Yes. That's the "workaround" for most cases, but my problem is specifically with the `transparency` use-case.
